tests/test_DataAugmenterInternally.py

```diff
@@ -0,0 +1,156 @@
+import os
+import sys
+import pytest
+import numpy as np
+import pandas as pd
+
+from scipy.stats import ks_2samp
+
+sys.path.append("zarnitsa/")
+
+from DataAugmenterInternally import DataAugmenterInternally
+
+"""
+Under the null hypothesis the two distributions are identical.
+If the K-S statistic is small or the p-value is high
+(greater than the significance level, say 5%),
+then we cannot reject the hypothesis that the distributions of
+the two samples are the same.
+Conversely, we can reject the null hypothesis if the p-value is low.
+"""
+
+N_TO_CHECK = 500
+
+
+@pytest.fixture
+def dataug_internally():
+    return DataAugmenterInternally()
+
+
+def test_augment_column(dataug_internally):
+    """
+    Augment column
+    """
+    pd_series = pd.Series([], dtype="float64")
+    try:
+        dataug_internally.augment_column_norm(pd_series)
+        assert False, "ValueError was throwed!"
+    except ValueError as e:
+        assert True
+
+
+def test_augment_column_permut(dataug_internally):
+    """
```

Comment:
again, don't mix different cases in test function's body and don't hardcode data into body, send it as args